test/test_train_mp_imagenet_fsdp.py

```diff
@@ -215,25 +222,57 @@ def train_imagenet():
 
   device = xm.xla_device()
   model = get_model_property('model_fn')()
-  # Wrap the model with FSDP
-  # You may wrap all, a subset, or none of the sub-modules with inner FSDPs
-  # - to implement ZeRO-2, wrap none of the sub-modules
-  # - to implement ZeRO-3, wrap all of the sub-modules (nested FSDP)
-  # - you may wrap sub-modules at different granularity (e.g. at each resnet
-  #   stage or each residual block or each conv layer).
+  # Automatic wrapping sub-modules with inner FSDP
+  auto_wrap_policy = None
+  auto_wrapper_callable = None
+  if FLAGS.auto_wrap_policy != "none":
+    if FLAGS.auto_wrap_policy == "size_based":
+      # auto-wrap all sub-modules with more than 1e6 parameters as an example
+      auto_wrap_policy = partial(
+          size_based_auto_wrap_policy, min_num_params=1e6)
```

Comment:
is this `min_num_params` config-able?

Reply:
Yes, this should be config-able. I added a new flag `--auto_wrap_min_num_params` to config it.